modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/definitions/display/context/ViewObjectDefinitionsDisplayContext.java

```diff
@@ -181,6 +205,16 @@ private boolean _hasAddObjectDefinitionPermission() {
 			ObjectActionKeys.ADD_OBJECT_DEFINITION);
 	}
 
+	private boolean _hasUpdateObjectDefinitionPermission() {
+		PortletResourcePermission portletResourcePermission =
+			_objectDefinitionModelResourcePermission.
+				getPortletResourcePermission();
+
+		return portletResourcePermission.contains(
+			_objectRequestHelper.getPermissionChecker(), null,
+			ObjectActionKeys.PUBLISH_OBJECT_DEFINITION);
```

Comment:
I realized I used the wrong permission for this validation. I will change this validation to use the update permission and push my changes.